orca/producer_test.go

```diff
@@ -143,8 +143,9 @@ func (s) TestProducer(t *testing.T) {
 	oobLis := newTestOOBListener()
 
 	lisOpts := orca.OOBListenerOptions{ReportInterval: 50 * time.Millisecond}
-	lao := &listenerAndOptions{listener: oobLis, opts: lisOpts}
-	r.InitialState(resolver.State{Addresses: []resolver.Address{setListenerAndOptions(resolver.Address{Addr: lis.Addr().String()}, lao)}})
+	lao := &listenerInfo{listener: oobLis, opts: lisOpts}
```

Comment:
Nit: switch this acronym now that object type is changed. Here and elsewhere.

Reply:
Done